mailpoet/assets/js/src/automation/integrations/mailpoet/steps/tag_added/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { __ } from '@wordpress/i18n';
+import { StepType } from '../../../../editor/store';
+import { LockedBadge } from '../../../../../common/premium_modal/locked_badge';
+import { PremiumModalForStepEdit } from '../../../../../common/premium_modal';
+
+const keywords = ['tag', 'tags', 'tag added', 'tags added'];
```

Comment:
I think in other places we've already made these translatable.

Reply:
6ed521f2e